src/Mvc/Mvc.Core/src/RedirectResult.cs

```diff
@@ -94,6 +123,14 @@ public string Url
             }
         }
 
+        /// <summary>
+        /// Gets the status code to redirect with.
+        /// </summary>
+        public int? statusCode
```

Comment:
Please use Pascal Case for properties: `StatusCode` with capital letter

Reply:
Also you can rewrite this as `public int? StatusCode => _statusCode;`

Reply:
Modified  as requested.